association-de-science-regionale-de-langue-francaise.csl

```diff
@@ -0,0 +1,482 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="fr-CA">
```

Comment:
If it's standard French, the locale code on `<style>` and `<locale>` below should be changed to "fr-FR". If Canadian French is correct, the title should end on "(French - Canada)" instead of "(French)".